packages/react-query-4/src/SuspenseInfiniteQuery.tsx

```diff
@@ -6,6 +6,22 @@ import {
   useSuspenseInfiniteQuery,
 } from './useSuspenseInfiniteQuery'
 
+/**
+ * We provide these components to clearly express what causes suspense at the same depth.
+ * `<SuspenseInfiniteQuery/>` serves to make `useSuspenseInfiniteQuery` easier to use in jsx.
+ * @see {@link https://suspensive.org/en/docs/react-query/SuspenseInfiniteQuery Suspensive Docs}
+ * @example
+ * ```tsx
+ * import { SuspenseInfiniteQuery } from '@suspensive/react-query'
+ *
+ * // You can use infiniteQueryOptions as props.
+ * <SuspenseInfiniteQuery {...infiniteQueryOptions()}>
+ *   {({ data, fetchNextPage }) => { // infiniteQueryResult can be used in children.
```

Comment:
```suggestion
 *   {({ data, fetchNextPage }) => {
```

I want to remove this, because infiniteQueryResult is not official word